### DISCUSSION TITLE
Lantmäteriverket Terrängkarta / Maanmittauslaitos Maastokartta har slutat fungera

### DISCUSSION THREAD

**Discussion by PeterWestoo** [johanberonius/custom-map-source.appspot.com#9] [General]
Hej! Den här fungerar inte i Guru Maps (sen i höstas). Kartan fungerar i andra "tredjepartsappar" så tror inte den är helt låst. /Peter

**Comment by johanberonius** [comment]
Webbservern kräver att HTTP-headern `Referer` är satt till `.maanmittauslaitos.fi` annars får man bara svaret `403 Forbidden`. Detta just för att sätta stopp för laddning av kartbilder från tredjepartsappar. Det här går att fejka när man testar lokalt men det går inte att tvinga andra appar att sätta ett fejkat värde.

Har du något exempel där det faktiskt fungerar?

Exempel på en länk till en kartruta:
https://karttamoottori.maanmittauslaitos.fi/maasto/wmts/1.0.0/maastokartta/default/WGS84_Pseudo-Mercator/7/37/71.png

Funkar inte att ladda den länken direkt men man kan testa med ett terminal-kommando:
```
curl -v --header "Referer: .maanmittauslaitos.fi" 'https://karttamoottori.maanmittauslaitos.fi/maasto/wmts/1.0.0/maastokartta/default/WGS84_Pseudo-Mercator/7/37/71.png'
```
